sdk/core/core-amqp/src/index.ts

```diff
@@ -39,3 +39,4 @@ export {
 } from "./util/utils";
 export { AmqpAnnotatedMessage } from "./amqpAnnotatedMessage";
 export { logger } from "./log";
+export * from "./internals";
```

Comment:
I really wanted to do `export * as Internals from "./internals"; but unfortunately api-extractor doesn't support it (yet).
https://github.com/microsoft/rushstack/issues/2412